core/variant/variant_parser.cpp

```diff
@@ -1075,7 +1075,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 				return ERR_PARSE_ERROR;
 			}
 
-			static HashMap<StringName, Variant::Type> builtin_types;
+			static HashMap<String, Variant::Type> builtin_types;
```

Comment:
For the record, I checked and confirmed that `token.value` used below should indeed be a String when `token.type` is `TK_IDENTIFIER`.